alicloud/resource_alicloud_cen_bandwidth_package.go

```diff
@@ -79,6 +79,27 @@ func resourceAlicloudCenBandwidthPackage() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"auto_renew": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+				ForceNew: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					return PayType(d.Get("charge_type").(string)) == PrePaid
```

Comment:
I think the line 89 should be `PayType(d.Get("charge_type").(string)) == PostPaid`